ee_tests/src/specs/page_objects/space_dashboard.page.ts

```diff
@@ -228,17 +152,19 @@ export class SpaceDashboardPage extends SpaceTabPage {
   wizardSidebar = element(by.xpath('.//*[contains(@class,\'wizard-pf-sidebar\')]'));
 
   spaceName: string;
-
+  header = new SpaceDashboardPageHeader();
```

Comment:
must be passed in the elementFinder so that it reads something like `header = new SpaceDashpardPageHeader(this.$('sub-element'))